at_secondary/at_persistence_secondary_server/lib/src/notification/at_notification.dart

```diff
@@ -128,7 +128,7 @@ class AtNotification {
     return 'AtNotification{id: $_id,fromAtSign: $_fromAtSign, '
         'notificationDateTime: $_notificationDateTime, '
         'toAtSign:$_toAtSign, notification:$_notification, '
-        'type:$_type, opType:$_opType, expiresAt:$_expiresAt : priority:$priority : notificationStatus:$notificationStatus : atValue:$atValue';
+        'type:$_type, opType:$_opType, expiresAt:$_expiresAt : priority:$priority : notificationStatus:$notificationStatus : atValue:$atValue}';
```

Comment:
I would like @kalluriramkumar to have a look at this `}` fix.

Reply:
In my opinion, it's the right fix. @kalluriramkumar please confirm.

Reply:
Yeah, fix look fine.